src/components/Tool/NavToolList/NavToolList.tsx

```diff
@@ -1,61 +1,153 @@
 import React, { Fragment, ReactElement } from 'react';
 import Icon from '@mdi/react';
+import Close from '@mui/icons-material/Close';
 import {
   Box,
+  IconButton,
   List,
   ListItem,
   ListItemIcon,
   ListItemText,
+  Slide,
   Theme,
+  Typography,
 } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
 import { makeStyles } from 'tss-react/mui';
+import { useGetToolNotificationsQuery } from 'src/components/Layouts/Primary/TopBar/Items/NavMenu/GetToolNotifcations.generated';
+import { ToolName } from 'src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu';
+import { useAccountListId } from 'src/hooks/useAccountListId';
 import { ToolsList } from '../Home/ToolList';
 import { Item } from './Item/Item';
 
 const useStyles = makeStyles()((theme: Theme) => ({
```

Comment:
I don't love that we're using `makeStyles` and styled components in the same component. Could you switch to styled components?